pyenv3wrapper.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+HOME_DIR=$(getent passwd "$(whoami)" | cut -d: -f6)
```

Comment:
Isn't this already provided via `$HOME`?

Reply:
Bazel commands, to make them isolated from the environment and thus reporducible, are executed under `env -` environment, i.e. in clean environment, where none of the env variables are set, including `$HOME` `$PATH` or anything else. That is why to find home directory we need to rely only on the core built-in commands (like `whoami`) and some trickery to get home dir path from that.

Reply:
Okay, good to know. TIL